Tasks/InstallAppleCertificateV2/Tests/L0.ts

```diff
@@ -11,7 +12,7 @@ describe('InstallAppleCertificate Suite', function () {
     after(() => {
     });
 
-    it('Defaults: install cert in temporary keychain', (done: MochaDone) => {
+    it('Defaults: install cert in temporary keychain', (done: Done) => {
```

Comment:
Nit: could you please make it as Egor did in https://github.com/microsoft/azure-pipelines-tasks/pull/13730/files for consistency?